website_user_login_redirect_custom/tests/test_website_login_redirect.py

```
@@ -0,0 +1,118 @@
+        # Reset to default state before each test
+        self.config_parameter.set_param(
+            "website_user_login_redirect_custom.enabled", "0"
+        )
+        self.config_parameter.set_param("website_user_login_redirect_custom.url", "/")
+
+    def test_redirect_disabled_by_default(self):
+        """Test that redirect is disabled by default"""
+        enabled = self.config_parameter.get_param(
+            "website_user_login_redirect_custom.enabled"
+        )
+        self.assertEqual(enabled, "0")
```

Ruff (0.14.0)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)